### ISSUE TITLE
Meta Modulator (Warps) LED color is probably wrong

### ISSUE THREAD

**Issue by AndrewBelt** [VCVRack/AudibleInstruments#4]
Could someone tell me approximately the colors used for the modes of the algorithm knob? HSL is preferred.

**Comment by tildebyte** [comment]
Maybe I'm misunderstanding the issue, but [warps ui source...](https://github.com/AndrewBelt/eurorack/blob/eb6c8e9a5728a109ebe559aa78c4023be0b710e8/warps/ui.cc#L44)
It looks like some assignment/interpolation happens further down in `Ui::Poll()`
It'll clearly take some codesurfing and calculation to get exact colors... but maybe some kind soul can shoot a video 👅

**Comment by AndrewBelt** [comment]
Yeah, if I had time I could figure it out, like I did with Warps.

**Comment by tildebyte** [comment]
I'd volunteer, but my c++ fu is negligible.
